hack/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -o errexit
+
+KUBECTL="${KUBECTL:-kubectl}"
+KUSTOMIZE="${KUSTOMIZE:-kustomize}"
+WITH_PROMETHEUS="${WITH_PROMETHEUS:-false}"
+IMG="${IMG:-quay.io/metallb/frr-k8s:latest}"
+LOGLEVEL="${LOGLEVEL:-info}"
+
+KUSTOMIZE_LAYER="default"
+if [ "$WITH_PROMETHEUS" == "true" ]; then
```

Comment:
how about having a deploy-prometheus makefile rule instead of moving everything to this bash script?
Things will become messy once we add helm too. Also, not a fan of duplicating the env variables if not required

Reply:
done